modules/order/src/EventSubscriber/OrderMailer.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Drupal\commerce_order\EventSubscriber;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Mail\MailManagerInterface;
+use Drupal\Core\Render\Markup;
+use Drupal\Core\Render\RenderContext;
+use Drupal\Core\Render\Renderer;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\state_machine\Event\WorkflowTransitionEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Listens to order transition events and sends emails.
+ */
+class OrderMailer implements EventSubscriberInterface {
```

Comment:
@bojanz I think OrderMailSubscriber. 

Maybe we should add another email that is for the `order_fulfillment ` workflow. And on `fulfill` we send "your order has been fulfilled and is completed."